### PR TITLE
Switch data aggregating cronjob in ocp , pass the baton to ocp4

### PR DESCRIPTION
**Pull request by harshad16**
Switch data aggregating cronjob in ocp , pass the baton to ocp4
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: #504

### CODE REVIEW

File: core/overlays/ocp4-stage/common/configmaps.yaml [hunk 1]
```diff
@@ -37,6 +37,9 @@ metadata:
 data:
   solvers: |
     solver-rhel-8-py36
+    solver-rhel-8-py38
+    solver-fedora-32-py37
```
Comment (goern) [inline core/overlays/ocp4-stage/common/configmaps.yaml:41]: all at once?!

Comment (harshad16) [reply core/overlays/ocp4-stage/common/configmaps.yaml:41]: we can switch off the fedora ones ?

Comment (goern) [reply core/overlays/ocp4-stage/common/configmaps.yaml:41]: ja please do so, and lets observe what the load and workflows do...

**Pull request merged**